doc/api/assert.md

```diff
@@ -2006,6 +2006,32 @@ argument, then `error` is assumed to be omitted and the string will be used for
 example in [`assert.throws()`][] carefully if using a string as the second
 argument gets considered.
 
+## `assert.snapshot(value, name)`
+
+<!-- YAML
+added: REPLACEME
+-->
+
+> Stability: 1 - Experimental
+
+* `value` {any} the value to snapshot
+* `name` {string} the name of snapshot.
+* Returns: {Promise}
+
+reads a snapshot from a file, and compares `value` to the snapshot.
+`value` is serialized with [`util.inspect()`][]
+If the value is not strictly equal to the snapshot,
+`assert.snapshot()` will return a rejected `Promise`
+with an [`AssertionError`][].
+
+If the snapshot file does not exist, the snapshot is written.
+
+In case it is needed to force a snapshot update,
+set `process.env.NODE_UPDATE_SNAPSHOT=1`;
```

Comment:
Why are we using an environment variable for this? Node has been trying to limit new environment variables in favor of CLI flags (which can also be passed via the `NODE_OPTIONS` environment variable).

Reply:
updating to use `--update-assert-snapshot`